src/lists/list-item.jsx

```diff
@@ -296,16 +296,18 @@ const ListItem = React.createClass({
   },
 
   _createTextElement(styles, data, key) {
-    const isAnElement = React.isValidElement(data);
-    const mergedStyles = isAnElement ?
-      Object.assign({}, styles, data.props.style) : null;
-
-    return isAnElement ? (
-      React.cloneElement(data, {
+    if (React.isValidElement(data)) {
+      let style = Object.assign({}, styles, data.props.style);
+      if (typeof data.type === 'string') { // if element is a native dom node
```

Comment:
I don't get it. Why do you think  we should call `prepareStyles` for a `<span />` element and not for a home made `<Bar />` ?
```jsx
const Bar = () => (
  <span id="foo">
    Primary text
  </span>
);
```